lib/elixir/lib/io/ansi.ex

```diff
@@ -43,140 +44,140 @@ defmodule IO.ANSI do
   end
 
   @doc "Sets foreground color."
-  @spec color(0..255) :: String.t
+  @spec color(0..255) :: String.t()
   def color(code) when code in 0..255, do: "\e[38;5;#{code}m"
 
   @doc ~S"""
   Sets the foreground color from individual RGB values.
 
   Valid values for each color are in the range 0 to 5.
   """
-  @spec color(0..5, 0..5, 0..5) :: String.t
+  @spec color(0..5, 0..5, 0..5) :: String.t()
   def color(r, g, b) when r in 0..5 and g in 0..5 and b in 0..5 do
-    color(16 + (36 * r) + (6 * g) + b)
```

Comment:
I do not think that the code formatter should remove the additional grouping brackets